frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelistedHealthChecks.ts

```diff
@@ -0,0 +1,6 @@
+export const whitelistedHealthChecks = new Set(['MON_DISK_LOW', 'MON_DOWN']);
```

Comment:
create an enum of the Health check and use those values to refer them.
```suggestion
enum HealthChecks = {MON_DISK_LOW = 'MON_DISK_LOW'}
```